explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -423,18 +430,28 @@ object TargetTabContents extends TwoPanels:
                   .get(targetId)
                   .map:
                     case Nonsidereal(_, _, _) =>
-                      (renderNonSiderealTargetEditor, "target-non-sidereal-controller")
+                      (renderNonSiderealTargetEditor, TargetTabControllerIds.Summary.id)
                     case Sidereal(_, _, _, _) =>
-                      (renderSiderealTargetEditor(resize, targetId), "target-sidereal-controller")
+                      (renderSiderealTargetEditor(resize, targetId),
+                       TargetTabControllerIds.Summary.id
+                      )
               case Right(obsIds)  =>
                 findAsterismGroup(obsIds, props.programSummaries.get.asterismGroups)
                   .map: asterismGroup =>
-                    (renderAsterismEditor(resize, obsIds, asterismGroup), "target-obs-controller")
+                    (renderAsterismEditor(resize, obsIds, asterismGroup),
+                     TargetTabControllerIds.AsterismEditor.id
+                    )
+
+      val justSummaryTiles: List[Tile] = List(
```

Comment:
So the fix is to always have the same number of tiles, and then react won't remount existing tiles when tiles are added?

Reply:
The remount happened because we changed the key of the `TileController`. Now we only use 2 keys: one for when the summary is shown and when for when it isn't.

We did this to prevent the tile controller tripping up when the displayed tiles changed (the saved information would go bonkers). This PR prevents that by still having the unused tiles in the controller, but hidden.